app/src/main/res/layout/fragment_feed.xml

```diff
@@ -13,8 +13,7 @@
         <android.support.v7.widget.RecyclerView
             android:id="@+id/feed_recycler_view"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:paddingTop="8dp" />
```

Comment:
Why?

Reply:
That's the ugly gap on top (same for others)